src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,8 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+    List<? extends Machine> get();
```

Comment:
We should parametrize MachineProducer as was said in README.md(3), from here there must be made more changes.